### PR TITLE
Add GitHub Actions

### PR DESCRIPTION
**Pull request by gracieflores**
Remove Travis and add GitHub Actions for testing.

### CODE REVIEW

**Comment by gracieflores** [comment]
@ldko @somexpert This is ready for review.

File: README.rst [hunk 1]
```diff
@@ -217,8 +218,30 @@ Viewing the logs
 Running the Tests
 """""""""""""""""
 
-To run the tests via Tox, use this command.
+To run the tests via Tox, use this command. If you are using podman-compose, swap the word docker with podman (see ``Developing with Podman and Podman-Compose`` below).
```
Comment (ldko) [inline README.rst:221]: For the three longer lines that are added in this README, I would break them up, so the lines are less than 100 characters.

**Review by ldko — APPROVED** [review]
I think this looks good. I would perhaps shorten the longer README lines by splitting them onto multiple lines.

**Review by somexpert — APPROVED** [review]
Looks good to me and looks like it's working as expected! :thumbsup:

**Pull request merged**